dev/AppNotifications/AppNotificationUtility.cpp

```diff
@@ -237,14 +240,57 @@ std::wstring Microsoft::Windows::AppNotifications::Helpers::GetDisplayNameBasedO
     return displayName;
 }
 
-// Placeholder
-HRESULT RetrieveAssetsFromProcess(_Out_ Microsoft::Windows::AppNotifications::Helpers::AppNotificationAssets& /*assets*/)
+inline wil::unique_hicon RetrieveIconFromProcess()
```

Comment:
Recommend moving this all to a seperate class called ShellUtility.cpp, .h

Reply:
Recommend not naming anything `Utility`. That plus `Helper` are trigger words for me :P

Functions here all seem related to localizable data - icons, etc. `ShellLocalization.h`?

Reply:
That sounds good to me!